src/main/java/com/exasol/containers/ExasolContainer.java

```diff
@@ -596,6 +604,28 @@ private void clusterConfigurationIsAvailable() {
         }
     }
 
+    private static final int MAJOR_DEPRECATED_VERSION = 6;
+    private static final int MINOR_DEPRECATED_VERSION = 2;
+
+    private void ThrowDBVersionNotSupportedException() {
+        throw new ContainerLaunchException(
+                ExaError.messageBuilder("E-ETC-13")
+                        .message("Exasol Database version " + MAJOR_DEPRECATED_VERSION + "." + MINOR_DEPRECATED_VERSION
+                                + " and lower are no longer supported in this version of Exasol Testcontainers.")
+                        .toString());
+    }
+
+    private void CheckClusterConfigurationForMinimumSupportedDBVersion() {
+        // do the check
+        String dbVersion = clusterConfiguration.getDBVersion();
+        String[] dbVersionSplit = dbVersion.split(Pattern.quote("."));
+        int majorVersion = Integer.parseInt(dbVersionSplit[0]);
+        int minorVersion = Integer.parseInt(dbVersionSplit[1]);
+        if (majorVersion <= MAJOR_DEPRECATED_VERSION && minorVersion <= MINOR_DEPRECATED_VERSION) {
```

Comment:
This would not match `5.9`. 
(since 9 > 2)

Reply:
```suggestion
        if ((majorVersion == MAJOR_DEPRECATED_VERSION && minorVersion <= MINOR_DEPRECATED_VERSION) || (majorVersion < MAJOR_DEPRECATED_VERSION && minorVersion <= MINOR_DEPRECATED_VERSION)) {
```

Reply:
🤦‍♂️